qiling/os/windows/dlls/kernel32/debugapi.py

```diff
@@ -47,3 +47,13 @@ def hook_OutputDebugStringW(ql, address, params):
     string = params["lpOutputString"]
     ql.os.stdout.write(string.encode())
     return 0
+
+#void OutputDebugStringA(
+#  LPCSTR lpOutputString
+#);
+@winapi(cc=STDCALL, params={
+    "lpOutputString": STRING
+})
+def hook_OutputDebugStringA(ql, address, params):
+    ql.os.stdout.write('OutputDebugStringA: "%s"' % (params['lpOutputString']))
```

Comment:
will it be better to make this two lines of ql.os.stdout.write into ql.drpint ?

Reply:
I'm not sure. I initially did have this implemented only as ql.dprint, but noticed that the existing OutputDebugStringW implementation only used ql.os.stdout.write, so I just changed mine to match it.

That doesn't seem correct though, since OutputDebugString will not actually emit any console input. I'll change both of them to use ql.dprint.

Reply:
the reson why ql.drpint or ql.nprint we are in control. we can choose to print or not to print. in this case should be nprint and not dprint.

Reply:
Changed to nprint only